start adding effects

JBaczuk opened the pull request:
Adds a pulse effect for testing when you click play in the frontend on a device

JBaczuk commented on api/colors.json:
```diff
@@ -0,0 +1,62 @@
+{
```
@YeonV I think we have this already in https://github.com/LedFx/ledfx_rewrite/blob/main/color/predefined.go

shauneccles commented:
Are we ready to merge this?

JBaczuk commented:
> Are we ready to merge this?

We can, but we should probably remove some redundant code, I'll do that.

carterpeel commented:
> > Are we ready to merge this?
> 
> We can, but we should probably remove some redundant code, I'll do that.

Are you able to fix the conflicts so we can merge this?

JBaczuk commented:
> > > Are we ready to merge this?
> > 
> > 
> > We can, but we should probably remove some redundant code, I'll do that.
> 
> Are you able to fix the conflicts so we can merge this?

Hi, sorry I've been busy I will try to take a look soon

JBaczuk commented:
@carterpeel Just went through and fixed all the conflicts. It looks like the effects module was removed, not sure why. So I removed all references to it.
